csodiaq/identifier/outputWritingFunctions.py

```diff
@@ -0,0 +1,72 @@
+import pandas as pd
+
+def format_output_line(libMetadata, queMetadata, matchMetadata):
+    return [
+        queMetadata["scan"],
+        queMetadata["precursorMz"],
+        libMetadata["peptide"],
+        libMetadata["proteinName"],
+        libMetadata["precursorMz"],
+        libMetadata["precursorCharge"],
+        matchMetadata["cosineSimilarityScore"],
+        libMetadata["identifier"],
+        queMetadata["peaksCount"],
+        libMetadata["peaks"],
+        matchMetadata["shared"],
+        matchMetadata["ionCount"],
+        queMetadata["CV"],
+        queMetadata["windowWidth"],
+        matchMetadata["maccScore"],
+        #matchMetadata["excludeNum"],
```

Comment:
I've made a TODO about this and will be asking Jesse about it. I expect this to be re-added to the output once I've parsed that out.